clusterloader2/pkg/prometheus/manifests/prometheus-serviceMonitorKubeProxy.yaml

```diff
@@ -12,9 +12,9 @@ spec:
       # collect them as often as in smaller clusters. For example if in 100 node cluster the test
       # takes 10min it's useful to gather metrics every 30s. On the other hand if the test takes
       # 10h collecting metrics every 5min is still good enough. The expression below should give us
-      # 30s interval for small clusters (up to 500 nodes), 2.5min interval for 2K node clusters and
-      # 5.5min interval for 5K node clusters.
-    - interval: {{MultiplyInt 30 (AddInt 1 (DivideInt .Nodes 500))}}s
+      # 30s interval for small clusters (up to 1000 nodes), 1.5min interval for 2K node clusters and
```

Comment:
Would it be possible to have:
30s if N<=1000
1m for bigger ones?

Or would it kill us?
I think I would prefer to have smaller intervals if possible.

Reply:
Hard to tell, the only way of finding out is trying it. 

Changed as you proposed, if it causes any issue I will change it back to what it was.